core/app/models/spree/order_contents.rb

```diff
@@ -16,6 +16,11 @@ def remove(variant, quantity = 1, options = {})
       after_add_or_remove(line_item, options)
     end
 
+    def remove_line_item(line_item, options = {})
+      line_item.destroy
```

Comment:
I feel like this should be either:

```ruby
line_item.destroy!
after_add_or_remove(line_item, options)
```

or

```ruby
line_item.destroy && after_add_or_remove(line_item, options)
```

Mainly because, if a user extends line items and breaks an `after_destroy` callback or something, I feel like this would help them debug.